frontend/src/redux/slices/cartSlice.js

```diff
@@ -86,58 +86,29 @@ const cartSlice = createSlice({
       state[type + 'Items'] = newCartItems;
       removeFromLocalStorage(info.id);
     },
-    decreaseCart(state, action) {
-      const decreaseInLocalStorage = (id) => {
-        let items = JSON.parse(localStorage.getItem(type)) || [];
 
-        items = items.map((item) =>
-          item.id === id
-            ? {
-                id: item.id,
-                cardQuantity: item.cardQuantity - 1,
-              }
-            : item,
-        );
-
-        updateLocalStorage(type, items);
-      };
-      const { info, type } = action.payload;
-
-      const itemIndex = state[type + 'Items'].findIndex(
-        (item) => item.id === info.id,
-      );
-      if (state[type + 'Items'][itemIndex].cardQuantity > 1) {
-        state[type + 'Items'][itemIndex].cardQuantity -= 1;
-      } else {
-        const newCartItems = state[type + 'Items'].filter(
-          (cartItem) => cartItem.id !== info.id,
-        );
-        state[type + 'Items'] = newCartItems;
-      }
-      decreaseInLocalStorage(info.id);
-    },
-    increaseCart(state, action) {
-      const increaseInLocalStorage = (id) => {
+    updateQuantityInCart(state, action) {
+      const updateQuantityLocalStorage = (id, amount) => {
```

Comment:
In my opinion, it would be better to extract this function for local storage to the top level